Makefile

```diff
@@ -154,6 +154,10 @@ dev-requirements:  ## Update dev-*requirements.txt files if pinned versions do n
 	pip-compile --allow-unsafe --generate-hashes --output-file requirements/dev-requirements.txt requirements/dev-requirements.in
 	pip-compile --allow-unsafe --generate-hashes --output-file requirements/dev-sdw-requirements.txt requirements/dev-sdw-requirements.in
 
+.PHONY: dev-buster-requirements
+dev-buster-requirements:  ## Update dev-*requirements.txt files if pinned versions do not comply with the dependency specifications in dev-*requirements.in
```

Comment:
Is there a reason this needs to be a separate target and can't just be part of `make dev-requirements`?

Reply:
Oh, I assumed it wasn't part of `make dev-requirements` because we'd like it gone already, so I made a different target that will be easy to spot when we start deleting Buster-specific code. Not a strong reason.